allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -207,16 +208,48 @@ def from_data_file(cls, data_file: EyeTrackingFile,
         dilation_frames : int, optional
              See EyeTracking.from_lims
         """
+        def _get_lost_frames(file_path: str) -> List[int]:
+            """
+            Get lost frames from the video metdata json
+            Must subtract one since the json starts indexing at 1
+
+            Parameters
+            ----------
+            file_path: str
+                Path to the metadat json
```

Comment:
Typo. Also please specify what metadata json you are referring to

Reply:
the `get_lost_frames` is no longer committed as you suggested.